<?xml version="1.0" encoding="UTF-8"?>
<ns2:test-suite xmlns:ns2="urn:model.allure.qatools.yandex.ru" start="1540208212536" stop="1540208272860">
	<name>Customer login Test </name>
	<title>Customer login Test </title>
	<test-cases>
		<test-case start="1540208212537" stop="1540208272859" status="failed">
			<name>Login as a customer</name>
			<title>Login as a customer</title>
			<description></description>
			<labels>
			</labels>
			<steps>
			</steps>
			<attachments>
				<attachment title="Screenshot" source="8153de27-d3be-4260-b94a-4a28ab63de78-attachment.png" type="image/png" size="296683"/>
			</attachments>
			<failure>
				<message>Error: Timeout - Async callback was not invoked within timeout specified by jasmine.DEFAULT_TIMEOUT_INTERVAL.</message>
				<stack-trace>Error: Timeout - Async callback was not invoked within timeout specified by jasmine.DEFAULT_TIMEOUT_INTERVAL.
    at Timeout._onTimeout (C:\Users\madhan\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4281:23)
    at ontimeout (timers.js:482:11)
    at tryOnTimeout (timers.js:317:5)
    at Timer.listOnTimeout (timers.js:277:5)</stack-trace>
			</failure>
		</test-case>
	</test-cases>
</ns2:test-suite>